webpack/scenes/Subscriptions/Manifest/SimpleContentAccess.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Switch, Level, LevelItem } from '@patternfly/react-core';
+import { OverlayTrigger, Tooltip, Icon } from 'patternfly-react';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { Col, ControlLabel } from 'react-bootstrap';
+
+const SimpleContentAccess = (props) => {
+  const {
+    canToggleSimpleContentAccess,
+    isSimpleContentAccessEnabled,
+    enableSimpleContentAccess,
+    disableSimpleContentAccess,
+    hideModal,
+  } = props;
+
+  const toggleSimpleContentAccess = () => {
+    if (isSimpleContentAccessEnabled) {
+      disableSimpleContentAccess();
+    } else {
+      enableSimpleContentAccess();
+    }
+  };
+
+  const handleChange = () => {
+    toggleSimpleContentAccess();
```

Comment:
I think this conversation was touched upon so let's chat about it before making the change I am about to suggest: When I toggle the modal immediately disappears, there is a moment where nothing happens, and then the progress bar modal appears. If we use 'await' here, then it should behave a little more smoothly. The modal will close once the task has been started, and the modal should pop up at that time because the reducer will have put it into the state.

Reply:
We can hide the modal when the task starts, In between the modal can be remain open and we can disable all the buttons.

Reply:
That sounds good in combination with removing the hideModal() functionality of the component per my below comment. Let's do it :)

Reply:
@jturel I have made the changes and we are hiding the modal when task is getting started. I have made the changes for refresh action also. We can make changes for upload and delete Manifest action also but because it has more than one modal open, it was looking weird to me to hide 2 modal in time of few seconds, So I have not made changes there. If you want I can also remove `hideModal` function call from those actions.